### ISSUE TITLE
[1.12.2 Creature Config Question] Unable to increase//decrease Spawn-Area Limit

### ISSUE THREAD

**Issue by Morpheus1101** [Lycanite/LycanitesMobs#429]
With the pack i have im swapping out def MC zombies to have cryptzombies during the day in combination with incontrol form mcjty, however being i need there to be quite a lot of them an hordes of them i need to be able to increase there spawn-Area limit which currently resets itself, do i need to change below to false or am i missing something?

>  "loadDefault": true,

**Comment by Lycanite** [comment]
Yup "loadDefault" must be set to false, each json config has this setting, when false it will just reset to defaults (so that I can make changes and have user config sauto update unless they've set this to false).
